src/application/router/routes.ts

```diff
@@ -195,6 +194,22 @@ const routes: RouteRecordRaw[] = [
       code: 404,
     },
   },
+  /**
+   * error page
+   */
+  {
+    path: '/error/:code',
+    component: ErrorPage,
+    meta: {
+      layout: 'fullpage',
+      pageTitleI18n: 'pages.error',
+      discardTabOnLeave: true,
+    },
+    props: route => ({
+      code: route.params.code,
+      customMessage: route.query.message,
```

Comment:
I dont see where `?message` is composed. And it's better to use a prop instead of adding a message to the URL.

Reply:
you want 'page not found' message right, so on error component it is getting that message from i18n based on error code, if you want custom message then we can pass it as query

Reply:
it is unused at the moment, so you can remove it.
When adding it in a future, it's better to pass custom message via props instead of URL